app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -515,11 +515,6 @@ public void applyTextFormat(int textFormatId) {
         _textFormat.getTextActions()
                 .setHighlightingEditor(_hlEditor)
                 .appendTextActionsToBar(_textActionsBar);
-        if (_textActionsBar.getChildCount() == 0) {
-            _textActionsBar.setVisibility(View.GONE);
-        } else {
-            _textActionsBar.setVisibility(View.VISIBLE);
-        }
```

Comment:
@gsantner Please note that I removed this existing code as it didn't seem to do anything useful.